docs/index.pug

```diff
@@ -0,0 +1,11 @@
+extends /layout/base.pug
+
+block content
+	main.col-md-24
+		h3 CKEditor 5 Documentation
+		hr
```

Comment:
I will remove this file. We can use [`readme.md`](https://github.com/ckeditor/ckeditor5/blob/master/README.md) file as the main page for the documentation. In the future, we will be able to change it.